assets/javascripts/discourse/templates/components/gamification-leaderboard-row.hbs

```diff
@@ -4,5 +4,11 @@
     {{avatar rank imageSize="large"}}
     <span class="user__name">{{rank.username}}</span>
   </div>
-  <div class="user__score">{{number rank.total_score}}</div>
+  <div class="user__score">
+    {{#if site.mobileView}}
```

Comment:
If it fits, I would opt to show the same value in mobile, i.e. `1.200` instead of `1.2k`.

Reply:
This was a suggestion that came out of the topic on dev. Unless you mean checking if its fits and then conditionally applying it? But I wouldn't know how to do that… 🤔

Reply:
Ok, that's fine.